.github/workflows/release.yml

```diff
@@ -1,157 +1,236 @@
 name: Wadm Release
-
 on:
   push:
-    tags: ["v*"] # Push events to matching v*, i.e. v1.0, v20.15.10
+    tags: 
+      - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
+      - "canary"
```

Comment:
By default a push to main should be `canary` and just overwrite the current version. That is basically what you have below, but we aren't planning on pushing just a plain canary tag